.github/workflows/test.yml

```diff
@@ -12,20 +12,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+## Runs the eslint and jest unit tests for every PR and push.
+
 name: Test
 on: [push, pull_request]
```

Comment:
You'll also need `on: workflow_call` for the Dependabot workflow to be able to call this one. That's why dependabot has been failing since merging #505.